my-app/src/Login/Login.js

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import axios from "axios";
+
+class Login extends React.Component {
+  state = {
+    username: "sam",
+    password: "pass"
+  };
+
+  render() {
+    return (
+      <>
+        <form onSubmit={this.handleSubmit}>
+          <div>
+            <label htmlFor="username" />
+            <input
+              value={this.state.username}
+              onChange={this.handleInputChange}
+              id="username"
+              type="text"
+            />
+          </div>
+          <div>
+            <label htmlFor="password" />
+            <input
+              value={this.state.password}
+              onChange={this.handleInputChange}
+              id="password"
+              type="password"
+            />
+          </div>
+          <div>
+            <button type="submit">Login</button>
+          </div>
+        </form>
+      </>
+    );
+  }
+
+  handleSubmit = event => {
+    event.preventDefault();
+
+    const endpoint = "http://localhost:5000/api/auth/login";
```

Comment:
```suggestion
    const endpoint = "http://localhost:6500/api/auth/login";
```
Your server is running on port 6500